helm/istio/charts/tracing/templates/jaeger-production-elasticsearch.yaml

```diff
@@ -22,7 +24,9 @@ spec:
       {{- end }}
 
   collector:
+    {{- if and .Values.jaeger.hub .Values.jaeger.tag }}
     image: {{ .Values.jaeger.hub }}/jaeger-collector:{{ .Values.jaeger.tag }}
```

Comment:
The image name should be jaeger-collector-rhel7, @objectiser can you please confirm?